CWP/papers/roadmap/authors/a2tex.py

```diff
@@ -0,0 +1,185 @@
+#! /usr/bin/env python
+#
+# Process CWP signatory list into a LaTeX author list
+#
+# The data source is expected to be made of 3 text files:
+#    - authors.txt: the list of signers in the following format: Lastname, Firstname - AffiliationKey (i) [- Role ]
```

Comment:
The footnote should always come at the end of the line, see the regexp at line 42.

Reply:
It was your original comment! I'll fix it!